packages/dart_frog_cli/lib/src/commands/dev/dev.dart

```diff
@@ -106,15 +111,31 @@ class DevCommand extends DartFrogCommand {
     _ensureRuntimeCompatibility(cwd);
 
     final port = io.Platform.environment['PORT'] ?? results['port'] as String;
+
     final dartVmServicePort = (results['dart-vm-service-port'] as String?) ??
         _defaultDartVmServicePort;
     final generator = await _generator(dartFrogDevServerBundle);
 
+    final hostname = results['hostname'] as String?;
+
+    late final io.InternetAddress? ip;
+    if (hostname == null) {
+      ip = null;
+    } else {
+      ip = io.InternetAddress.tryParse(hostname);
+      if (ip == null) {
+        logger.err('Invalid hostname "$hostname": must be a valid '
+            'IPv4 or IPv6 address.');
```

Comment:
Nit: Use of long string instead. 
```suggestion
        logger.err('''Invalid hostname "$hostname": must be a valid IPv4 or IPv6 address.''');
```

> The same NIT applies to the dev_server_domain.dart